packages/xo-server/src/api/host.js

```diff
@@ -23,6 +31,7 @@ set.description = 'changes the properties of an host'
 
 set.params = {
   id: { type: 'string' },
+  iscsiIqn: { type: ['string'], optional: true },
```

Comment:
You can remove the array, it's not necessary for a single type, look at the other params.